16-google/readme.md

```diff
@@ -0,0 +1,17 @@
+# ReadMe 16-Google
+
+## Incident Report
+
+https://status.cloud.google.com/incident/cloud-networking/19009
+
+## Summary
+
+Multiple clusters and supporting infrastructure being de-sechuled due to a defect in an upgrade script. This failure caused packet loss, client request timeouts, and increased latency.
+
+## Expectations
+
+Expecting near-100% success rate of events being returned by servers. Instead, the failed cluster management software begins showing packet failures, and eventually the events caught in the congested network timeout. Expect the output to display network congestion, higher latency, and event failures.
```

Comment:
Can you change this section to be similar to ones that I've done in incidents 1-5? i.e. a (~2-3) bulleted list of what stat changes we should be seeing during the run to verify that the incident still works if we ever have to update quartermaster.